input/config/config.yaml

```diff
@@ -0,0 +1,10 @@
+norovirus_url: https://raw.githubusercontent.com/jonathonmellor/norovirus-nowcast/refs/heads/main/outputs/data/cases_with_noise.csv
```

Comment:
Can you do this in a nested way to reduce duplication and enforce some kind of format? i.e norovirus an then pest all the things vs duplicating with prefixes. 

In some efficient future this would let you create new experiments on the fly by adding a new top level (and letting the code loop over them). I think its fine if you don't want to do that here as its a set of preplanned experiments.